Task3.1/Task3.1/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace задание3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double y;
+            Console.WriteLine("enterx =");
+            Console.WriteLine(value: y = conclusion) ;
+            Console.ReadKey();
+        }
+        static double conclusion
+        {
+            get
+            {
+                double y;
+                var x = double.Parse(Console.ReadLine());
+                return y = 1 + Math.Sqrt((Math.Pow(x, 2) - 1) / (Math.Pow(x, 2) + 1));
```

Comment:
Для возведения в квадрат проще использовать умножение
```suggestion
                return y = 1 + Math.Sqrt((x * x - 1) / (x * x + 1));
```